crates/re_viewer/data/quick_start_guides/how_does_it_work.md

```diff
@@ -0,0 +1,3 @@
+### How does it work?
+
+TBC
```

Comment:
Link to the relevant issue instead. Remember that anything merged is something that a user will stumble over!